app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    split_command = command.split()
+
+    if split_command[1] == split_command[2]:
```

Comment:
Validate that split_command have 3 parameters
Also please read checklist! You must check that command is cp